meet_balls_app/enums.py

```diff
@@ -0,0 +1,17 @@
+from enum import Enum
+
+
+class BallGame(Enum):
+    SOCCER = 'Soccer'
+    BASKETBALL = 'Basketball'
+    VOLLEYBALL = 'Volleyball'
+    BASEBALL = 'Baseball'
+    TENNIS = 'Tennis'
+    RUGBY = 'Rugby'
+    GOLF = 'Golf'
+    CRICKET = 'Cricket'
+    HANDBALL = 'Handball'
+
+    @classmethod
+    def BALL_GAME_CHOICES(cls):
```

Comment:
function name should be in lower case like the convention.
Also add new line at the end of this file.

Reply:
Thank you for the comment, it's fixed now.

Reply:
I am a bit confused, according to Amit you have implemented BallGame with TextChoices
In this implementation, you receive the choices function directly from Django, so there is no need to implement it

But here I see that it is implemented as an Enum
Can you clarify the final implementation for me?

Reply:
> I am a bit confused, according to Amit you have implemented BallGame with TextChoices In this implementation, you receive the choices function directly from Django, so there is no need to implement it
> 
> But here I see that it is implemented as an Enum Can you clarify the final implementation for me?

I thought Amit refered to the comment where you hinted that we want to store only the tag.value in the database, so to make it work i figured that for each tag.value we display "Soccer", "Soccer" for example. 

I am not really sure myself what's the best way to declare this enum.